### PR TITLE
added disabling method for ir export button

### PR DESCRIPTION
**Pull request by AntonStepanof**
fixes https://github.com/OHDSI/Atlas/issues/1879

### CODE REVIEW

File: js/pages/incidence-rates/components/iranalysis/components/results.js [hunk 1]
```diff
@@ -86,6 +86,10 @@ define([
 			});
 
 			this.executionDisabledReason = ko.computed(() => this.dirtyFlag().isDirty() ? constants.disabledReasons.DIRTY : constants.disabledReasons.ACCESS_DENIED);
+
+			this.disableExportAnalysis = ko.pureComputed(() => {
+				return this.dirtyFlag().isDirty() || this.sources().filter(s => s.info() != null).length === 0;
```
Comment (wivern) [inline js/pages/incidence-rates/components/iranalysis/components/results.js]: pls, use strict comparison, but `!!s.info()` looks better in such case,
moreover why not smth like `this.sources.some(s => !!s.info())`?

File: js/pages/incidence-rates/components/iranalysis/components/results.js [hunk 2]
```diff
@@ -88,7 +88,7 @@ define([
 			this.executionDisabledReason = ko.computed(() => this.dirtyFlag().isDirty() ? constants.disabledReasons.DIRTY : constants.disabledReasons.ACCESS_DENIED);
 
 			this.disableExportAnalysis = ko.pureComputed(() => {
-				return this.dirtyFlag().isDirty() || this.sources().filter(s => s.info() != null).length === 0;
+				return this.dirtyFlag().isDirty() || !this.sources().some(si => si.info() && si.info().executionInfo.status === "COMPLETE");
```
Comment (wivern) [inline js/pages/incidence-rates/components/iranalysis/components/results.js]: There is a constant for COMPLETE status

Comment (AntonStepanof) [reply js/pages/incidence-rates/components/iranalysis/components/results.js]: done

File: js/pages/incidence-rates/components/iranalysis/components/results.js (hunk 1 above)
Comment (AntonStepanof) [reply js/pages/incidence-rates/components/iranalysis/components/results.js]: updated

**Comment by AntonStepanof** [comment]
@pavgra Cannot reproduce error with pretty the same settings:
Screenshot 2019-06-12 at 00 21 54

**Comment by pavgra** [comment]
My bad, I haven't checked out the appropriate WebAPI branch

**Pull request merged**